.github/workflows/build_only.yml

```diff
@@ -13,20 +13,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['7.4', '8.0']
+        php-versions: ['7.4', '8.0', '8.1', '8.2']
```

Comment:
Probably 7.4 + 8.2 is enough, but this is fine too. Up to you.

Reply:
I'll keep it like that in case someone is using 8.0. It's not really hard to maintain them so I'll merge as-is.